bundles/org.openhab.binding.opensprinkler/src/main/java/org/openhab/binding/opensprinkler/internal/OpenSprinklerBindingConstants.java

```diff
@@ -26,169 +26,26 @@
     public static final String BINDING_ID = "opensprinkler";
 
     // List of all Thing ids
-    public static final String OPENSPRINKLER = "http";
-    public static final String OPENSPRINKLERPI = "pi";
+    public static final String HTTP_BRIDGE = "http";
+    public static final String STATION = "station";
+    public static final String DEVICE = "device";
+    public static final String PI = "pi";
```

Comment:
```suggestion
    public static final String PI_BRIDGE = "pi";
```

and maybe `_THING` on `STATION` and `DEVICE` to keep it consistent.

Reply:
Agreed!